.storybook/common.ts

```diff
@@ -83,7 +88,7 @@ export function getProducts(quantity: number): ProductModel[] {
   return products;
 }
 
-export const autocompleteResults = [
+export const autocompleteResults: AutocompleteResultGroup<AutocompleteSearchTermItem>[] = [
```

Comment:
Do we expect `AutocompleteResultGroup`s to contain something other than `SearchTermItem`s?

Reply:
I think the idea here was that it could be extensible: if a client wanted a different structure for their items because they're using their own components, they could do so easily.